modules/uploads/config.ts

```diff
@@ -0,0 +1,19 @@
+export interface Config {
+	maxUploadSize: UploadSize;
+	maxFilesPerUpload: number;
+	s3: S3Config;
+}
+
+type Units = "b" | "kb" | "mb" | "gb" | "tb" | "kib" | "mib" | "gib" | "tib";
+
+export type UploadSize = {
+	[unit in Units]: Record<unit, number>;
+}[Units];
+
+export interface S3Config {
+	bucket: string;
+	region: string;
+	accessKeyId: string;
+	secretAccessKey: string;
```

Comment:
Needs to be env vars. Should optionally override env var name that gets read. See sendgrid module.